### PR TITLE
initial implementation

### PR DESCRIPTION
**Pull request by alphmth**
#164

### CODE REVIEW

**Comment by jaredwray** [comment]
@alphmth - let me know if you need any help with this as it would be great to add to the mongo storage adapter!

**Comment by alphmth** [comment]
If you reviewed commits, please let me know if any change is needed.

**Comment by jaredwray** [comment]
Will review once the tests are passing

**Comment by alphmth** [comment]
@jaredwray please review

File: packages/mongo/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
 	"description": "MongoDB storage adapter for Keyv",
 	"main": "src/index.js",
 	"scripts": {
-		"test": "xo && nyc ava",
+		"test": "xo --fix && nyc ava",
```
Comment (jaredwray) [inline packages/mongo/package.json]: can you please remove this as it is not a standard on the project.

**Review by jaredwray — COMMENTED** [review]
just get that update in and we are good to go.

**Pull request merged**